### PR TITLE
[Feat] Supporting f32 and f64 for range

### PR DESCRIPTION
**Pull request by JulesGuesnon**

### CODE REVIEW

File: garde/src/rules/range.rs [hunk 1]
```diff
@@ -77,7 +77,7 @@ macro_rules! impl_for_int {
     };
 }
 
-impl_for_int!(u8, u16, u32, u64, usize, u128, i8, i16, i32, i64, isize, i128);
+impl_for_int!(u8, u16, u32, u64, usize, u128, i8, i16, i32, i64, isize, i128, f32, f64);
```
Comment (jprochazk) [inline garde/src/rules/range.rs:80]: Could you also add a `f32` field to this test here: 
https://github.com/jprochazk/garde/blob/1c2a13f3e31e269ea84322ea8b281222ea0c0aaa/garde/tests/rules/range.rs#L3-L10

**Review by jprochazk — CHANGES_REQUESTED** [review]
Thanks! Seeing as floating point types are a bit different from integer types, I'd just like to see a test for this so we don't break it in the future.

File: garde/src/rules/range.rs (hunk 1 above)
Comment (JulesGuesnon) [reply garde/src/rules/range.rs:80]: Hey sorry for the delay, I updated the test and the snapshot

**Review by jprochazk — APPROVED** [review]
Thank you!

**Comment by jprochazk** [comment]
Released in [v0.16](https://github.com/jprochazk/garde/releases/tag/v0.16.0)

**Comment by randomairborne** [comment]
Was this removed in 0.17? I was trying to validate floats and the Validate macro does complain about f32 not supporting the Apply function.

**Comment by jprochazk** [comment]
https://github.com/jprochazk/garde/blob/8f0091bd04772b96fb9cfe9de92dd45b49552627/garde/src/rules/range.rs#L80

Please post the full error message in an issue

**Comment by randomairborne** [comment]
Will do. Just wanted to ensure I wasn't going crazy.

**Comment by randomairborne** [comment]
Update: I was going crazy. When trying to update garde 0.16 to 0.17 I typed in 0.7, and somehow didn't catch that until just now as I was writing up the issue. My apologies.

**Pull request merged**